### PR TITLE
Add support for other providers to use non-default host and port [clean]

### PR DESCRIPTION
**Pull request by BeyondEvil**
Pushed a clean one instead. Hope that's OK?

Old one:  #111

### CODE REVIEW

File: docs/user_guide.rst [hunk 1]
```diff
@@ -390,6 +390,14 @@ the ``--capability`` command line arguments. See the
 `test options <http://testingbot.com/support/other/test-options>`_
 for full details of what can be configured.
 
+Local tunnel
+____________
```
Comment (davehunt) [inline docs/user_guide.rst]: Let's bump this up a level so it's alongside "Running tests".

File: docs/user_guide.rst [hunk 2]
```diff
@@ -390,6 +390,14 @@ the ``--capability`` command line arguments. See the
...
+
+To run the tests using `TestingBots local tunnel <https://testingbot.com/support/other/tunnel>`_
```
Comment (davehunt) [inline docs/user_guide.rst]: Nit: TestingBot's (note the apostrophe). Alternatively 'using a local TestingBot tunnel'.

File: docs/user_guide.rst [hunk 3]
```diff
@@ -390,6 +390,14 @@ the ``--capability`` command line arguments. See the
...
+you'll also need to set the ``--host`` and ``--port`` command line arguments::
+
+    --host localhost --port 4445
```
Comment (davehunt) [inline docs/user_guide.rst]: I think we should either give a full command line here, or just skip the example.

**Comment by BeyondEvil** [comment]
@davehunt Done! ^_^

**Pull request merged**